kubernetes/akvorado/templates/akvorado-inlet-deployment.yaml

```
@@ -23,7 +23,7 @@ spec:
     spec:
       initContainers:
         - name: wait-for-orchestrator
-          image: {{ include "common.images.image" ( dict "imageRoot" .Values.init.image ) }}
+          image: {{ include "common.images.image" ( dict "imageRoot" .Values.init.image "global" .Values.global) }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)